### PR TITLE
feat: database migration

### PR DESCRIPTION
**Pull request by HarisChoudhary**

### CODE REVIEW

File: cmd/database-migrator/main.go [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+	_ "github.com/threeport/threeport/cmd/database-migrator/migrations"
+	"github.com/threeport/threeport/pkg/api-server/v0/database"
+	cli "github.com/threeport/threeport/pkg/cli/v0"
+	log "github.com/threeport/threeport/pkg/log/v0"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+var (
+	DB_HOST     = "localhost"
```
Comment (HarisChoudhary) [inline cmd/database-migrator/main.go:25]: unify the db details in variables

File: cmd/tptctl/cmd/update.go [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+)
+
+var updateImageTag string
+
+// UpCmd represents the create threeport command
+var UpdateCmd = &cobra.Command{
+	Use:          "update",
+	Example:      "tptctl update --image-tag=v0.5",
```
Comment (HarisChoudhary) [inline cmd/tptctl/cmd/update.go:29]: change to upgrade

File: cmd/tptctl/cmd/update.go [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+	Short:        "Updates the deployment of the Threeport control plane",
```
Comment (HarisChoudhary) [inline cmd/tptctl/cmd/update.go:30]: upgrade the threeport control plane version

File: cmd/tptctl/cmd/update.go (hunk 2 above)
Comment (HarisChoudhary) [inline cmd/tptctl/cmd/update.go:29]: change image-tag version

Comment (HarisChoudhary) [inline cmd/tptctl/cmd/update.go:29]: Asana task for validation image tags for tptclt update

File: cmd/tptctl/cmd/update.go [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+	Long:         `Updates the deployment of the Threeport control plane.`,
```
Comment (HarisChoudhary) [inline cmd/tptctl/cmd/update.go:31]: Add note that you use a valid version e.g. v0.5.0.

File: cmd/tptctl/cmd/update.go [hunk 5]
```diff
@@ -0,0 +1,282 @@
...
+	SilenceUsage: true,
+	PreRun:       commandPreRunFunc,
+	Run: func(cmd *cobra.Command, args []string) {
```
Comment (HarisChoudhary) [inline cmd/tptctl/cmd/update.go:34]: Add regex validation for version

File: cmd/tptctl/cmd/upgrade_control_plane.go [hunk 6]
```diff
@@ -0,0 +1,283 @@
...
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+)
+
+var updateImageTag string
+
+// UpCmd represents the create threeport command
+var UpgradeControlPlaneCmd = &cobra.Command{
+	Use:     "control-plane",
+	Example: "tptctl upgrade control-plane --version=v0.5",
```
Comment (HarisChoudhary) [inline cmd/tptctl/cmd/upgrade_control_plane.go:29]: change to valid

**Review by lander2k2 — APPROVED** [review]

**Review by randalljohnson — APPROVED** [review]

**Pull request merged**